storage-proofs-update/src/vanilla.rs

```diff
@@ -850,43 +886,21 @@ where
             })?;
         new_replica_data.flush()?;
 
-        let (configs, replica_config) = split_config_and_replica(
-            tree_r_last_new_config.clone(),
-            new_replica_path.to_path_buf(),
-            base_tree_nodes_count,
-            tree_count,
-        )?;
-
         // Open the new written replica data as a DiskStore.
         let new_replica_store: DiskStore<TreeRDomain> =
             DiskStore::new_from_slice(nodes_count, &new_replica_data[0..])?;
 
-        let mut start = 0;
-        let mut end = base_tree_nodes_count;
-
-        for (i, config) in configs.iter().enumerate() {
-            let current_data: Vec<TreeRDomain> = new_replica_store.read_range(start..end)?;
-
-            start += base_tree_nodes_count;
-            end += base_tree_nodes_count;
-
-            info!(
-                "building base tree_r_last with CPU {}/{}",
-                i + 1,
-                tree_count
-            );
-            LCTree::<TreeRHasher, TreeR::Arity, U0, U0>::from_par_iter_with_config(
-                current_data,
-                config.clone(),
-            )?;
-        }
+        // This argument is currently unused by this invocation, but required for the API.
+        let mut unused_data = Data::empty();
```

Comment:
Maybe make the argument `Option` if it is not used in some cases?

Reply:
I had tried that before, but ran into rust move issues :sweat_smile: